docs/reverse_proxy.md

```diff
@@ -139,3 +139,10 @@ client IP addresses are recorded correctly.
 Having done so, you can then use `https://matrix.example.com` (instead
 of `https://matrix.example.com:8448`) as the "Custom server" when
 connecting to Synapse from a client.
+
+
+## Health check endpoint
+
+Synapse exposes a `/health` endpoint on every configured HTTP listener that
+always returns 200 OK (and doesn't get logged) for use by reverse proxies as a
+health check endpoint.
```

Comment:
I find this a bit confusing, the following might be clearer? Not sure it is an improvement.

```suggestion
Synapse exposes a health check endpoint for use by reverse proxies.
Each configured HTTP listener has a `/health` endpoint which always returns
200 OK (and doesn't get logged).
```

Maybe I'm just being a bit thick, but this means that if I have Synapse running on localhost:8080 I would be able to do the health check at: http://localhost:8080/health, correct? It isn't http://localhost:8080/_matrix/client/r0/health?

Reply:
Yup, its http://localhost:8080/health